downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -144,6 +144,22 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
            sample:main(sample.bal:3)
     ```
 
+- Fixed a bug that resulted in a compiler crash when attempting to ignore the error cause using `_` in an error binding pattern.
```

Comment:
```suggestion
- Fixed a bug that resulted in a compiler crash when attempting to ignore the error caused by using `_` in an error binding pattern.
```

Reply:
error cause is a part of an error: https://ballerina.io/learn/by-example/error-cause/ should we put `error cause` instead if it is misleading?

Reply:
Shall we add a comma after the word `cause`?